src/components/editor/faultTree/Editor.tsx

```diff
@@ -146,6 +148,17 @@ const Editor = ({setAppBarName}: DashboardTitleProps) => {
         })
     }
 
+    const handleCutSetAnalysis = () => {
+        console.log(`handleCutSetAnalysis(${faultTree.iri})` )
```

Comment:
```suggestion
```

Reply:
I do not consider it a good idea to include console logs.

Reply:
resolved